grid-packages/ag-grid-docs/documentation/src/pages/about.tsx

```diff
@@ -1,370 +1,380 @@
-                                that
-                                this thing had legs - and the idea of <strong>AG Grid Enterprise</strong> took seed in
-                                Niall’s
-                                mind. A little over one year after the project started the first commercial version was
-                                launched.
+                                that this thing had legs - and the idea of <strong>AG Grid Enterprise</strong> took seed
+                                in Niall’s mind. A little over one year after the project started the first commercial
+                                version was launched.
                             </p>
                             <p>
                                 Today, <strong>AG Grid</strong> is a self-funded, bootstrapped company with thousands of
                                 customers globally. Our product has resonated in the market - as our users face the same
                                 challenges Niall did - and this has been central to our rapid growth. And we’re not
-                                stopping
-                                here: we are working on the next great features to continue our mission.
+                                stopping here: we are working on the next great features to continue our mission.
                             </p>
                         </article>
                     </section>
                     <section>
                         <h2>Our Principles</h2>
                         <article>
-                            <p>We believe that a datagrid should be agnostic to the framework that developers choose.
-                                This
-                                allows flexibility and future-proofs your development. This is also where the 'ag'
+                            <p>
+                                We believe that a datagrid should be agnostic to the framework that developers choose.
+                                This allows flexibility and future-proofs your development. This is also where the 'ag'
                                 in <strong>AG Grid </strong>
-                                comes from. </p>
-                            <p>Our experience is in building Enterprise applications: we know that the datagrid is at
-                                the core
-                                of an Enterprise application, and needs to deliver performance and a rich feature
-                                set. </p>
-                            <p>We give away what others charge for. <strong>AG Grid Community</strong> provides all of
-                                the
-                                features of our competition. We only charge when we go above and beyond, with features
-                                that
-                                other grids don’t provide. </p>
+                                comes from.{' '}
```

Comment:
Don't think the space at the end of these paragraphs are necessary?

```suggestion
                                comes from.
```